guides/advanced_guides/faceted_search.md

```diff
@@ -0,0 +1,192 @@
+# Faceted Search
+
+**Faceted search** is a feature provided out-of-the-box by MeiliSearch. Faceting allows classifying search results into categories that are called **facets**.
+
+> For a movie, its title, its director, or its genre can be used as facets.
+
+A faceted search system provides users with a simple way to narrow down search results by selecting facets. A faceted navigation system is an **intuitive interface to display and navigate through content**. The facets are placed in the UI as filters which users can apply to refine the results in real-time.
+When users perform a search, they are presented with a list of results and a list of facets (i.e., categories) as below:
+
+![Amazon UI](/amazon-facets.png)
+> Faceted navigation on Amazon: facets are displayed on the left column.
+
+## How does it work?
+
+Faceted search, also known as faceted navigation, is a technique that combines traditional search with a **faceted classification of items**.
+Setting categorical document attributes as "facet" enables efficient filtering within the different categories. Such categorical attributes are, for example, movie genre, director, or language.
+Besides, faceting is a powerful feature that enables to build an intuitive navigation interface.
+
+Both faceting and filtering help drill down into a subset of search results. However, **faceting differs from [filtering](/guides/advanced_guides/filtering.md)**.
+
+- **Filters** exclude some results based on criteria. They allow users to narrow down a set of documents to only those matching these chosen criteria. In other words, filtering is used to filter the returned results by adding constraints.
+- **Facets**, on the other hand, are used to categorize the data into subsets that will be searched upon: they reduce the number of documents to process.
+
+Faceting and filtering aim at being complementary;  facets narrows down the set of documents to be searched upon, while filters reduce the number of documents coming out of a search.
+
+## Setting Up Facets
+
+Document attributes to use as facets **must be declared at indexing time**.
+
+You can set up facets **through the API** via the [global settings route](/references/settings.md#update-settings).
+Attributes for which to enable faceting are defined in the `attributesForFaceting` parameter. This parameter accepts an array of strings that specifies which attributes must be used as facets, defaulting to `null`:
+
+`attributesForFaceting=[<Attribute>, ...]`
+
+- `[<Attribute>, ...]` (Array of strings, defaults to `null`)
+
+  An array of strings that contains the attributes to use as facets.
+
+::: warning
+
+Only fields of data type **string** or **array of strings** can be set up as facets.
```

Comment:
It is not strictly disallowed as meilisearch does not enforce schema, it will just not work. One should not expect meilisearch to warn him if he uses bad params for faceting

```suggestion
Only fields of data type **string** or **array of strings** used for faceting.
```